x-pack/plugins/security_solution/public/detections/containers/detection_engine/rules/validate_rule_actions_field/validate_rule_actions_field.ts

```diff
@@ -59,34 +51,3 @@ export const validateRuleActionsField =
       };
     }
   };
-
-export const getSchema = ({
```

Comment:
moved to `x-pack/plugins/security_solution/public/detections/components/rules/step_rule_actions/get_schema.ts`